packages/contracts/contracts/BasicTokenMock.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.5.1;
```

Comment:
What's this contract for?

Reply:
Ah am using this contract as the ERC20 the deposit contract.